catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/accesscontrol/AccessControlPolicyPlugin.java

```diff
@@ -104,9 +124,64 @@ public PolicyResponse processPostQuery(Result input, Map<String, Serializable> p
     return new PolicyResponseImpl(Collections.emptyMap(), getPolicy(metacard));
   }
 
+  private boolean isUserOnlyRemoved(Metacard prev, Metacard updated, String attribute) {
+    List<String> prevValues = new ArrayList<>(getValuesOrEmpty(prev, attribute));
+    List<String> updatedValues = getValuesOrEmpty(updated, attribute);
+    if (prevValues.size() > updatedValues.size()) {
+      prevValues.removeAll(updatedValues);
+      return prevValues.size() == 1 && prevValues.contains(subjectSupplier.get());
+    }
+    return false;
+  }
+
+  private boolean isOnlyUserRemovedFromAccessControlAttribute(
+      Metacard prev, Metacard updated, String attribute) {
+    if (!attributeHasChanged(prev, updated, attribute)) {
+      return true;
+    }
+    return (Security.ACCESS_ADMINISTRATORS.equals(attribute)
+            || Security.ACCESS_INDIVIDUALS.equals(attribute)
+            || Security.ACCESS_INDIVIDUALS_READ.equals(attribute))
+        && isUserOnlyRemoved(prev, updated, attribute);
+  }
+
+  private boolean onlyUserAccessControlRemoved(Metacard prev, Metacard updated) {
```

Comment:
✏️ `isOnlyUserAccessControlRemoved` for the method name, more consistent with the one above?